src/main/java/de/tum/cit/aet/artemis/athena/domain/ModuleType.java

```diff
@@ -0,0 +1,5 @@
+package de.tum.cit.aet.artemis.athena.domain;
+
+public enum ModuleType {
+    FEEDBACK_SUGGESTIONS, PRELIMINARY_FEEDBACK
+}
```

Comment:
Is this module type, or should we call it somehow something else? Because we don't call this a module type on Athena right?